src/SprykerEco/Service/Computop/ComputopConfig.php

```diff
@@ -25,7 +25,7 @@ class ComputopConfig extends AbstractBundleConfig
      */
     public function isMacRequired($method)
     {
-        return in_array($method, $this->get(ComputopConstants::RESPONSE_MAC_REQUIRED, []));
+        return in_array($method, (array) $this->get(ComputopConstants::RESPONSE_MAC_REQUIRED, []));
```

Comment:
you can remove the default then

Reply:
Without `default`, an `Exception` will be thrown:
```
        if (!static::hasValue($key)) {
            throw new Exception(sprintf('Could not find config key "%s" in "%s"', $key, __CLASS__));
        }
```
…and I see that `mac` can be optional — method is named `isMacRequired()`